src/engine/ruleset/decoders/sophos/utm/sophos-utm.yml

```diff
@@ -0,0 +1,183 @@
+---
+name: decoder/sophos-utm/0
+
+metadata:
+  module: Sophos
+  title: Sophos-Utm logs decoder
+  version: [17.5.x, 18.0.x]
+  description: Decoder for Sophos Utm
+  author:
+    name: Wazuh Inc. info@wazuh.com
+    date: 2023-01-12
+  references:
+    - https://www.sophos.com/en-us/support/documentation/sophos-utm
+
+sources:
+  - decoder/queue-localfile/0
```

Comment:
Syslog supports syslog connections and could connect to Wazuh's syslog server, we should implement this in the parent decoders.
https://docs.sophos.com/nsg/sophos-utm/utm-on-aws/9.708/help/en-us/Content/utm/utmAdminGuide/LoggingSettingsRemoteSyslogServer.htm